### PR TITLE
Misc docs updates

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/features/working-with-xaml-hot-reload.md [hunk 1]
```diff
@@ -195,10 +382,10 @@ The output window in Visual Studio has an output named `Uno Platform` in its dro
 - When a file is reloaded, XAML parsing errors will appear in the application's logs, on the device or in the browser.
 - If there are multiple versions of the Uno.WinUI Package present in the solution, the newest will be used, regardless of the started application
 - For `net8.0-windows10.xx`:
+  - Ensure that the `net8.0-windows10.xxx` target framework **is selected in the top-left dropdown list of the XAML editor**. Any other selected platform will break hot reload.
```
Comment (morning4coffe-dev) [inline doc/articles/features/working-with-xaml-hot-reload.md:385]:
```suggestion
  - Ensure that the `net8.0-windows10.xxx` target framework **is selected in the top-left dropdown list of the XAML editor**. Selecting any other platform will break hot reload.
```

File: doc/articles/features/working-with-xaml-hot-reload.md [hunk 2]
```diff
@@ -82,17 +272,14 @@ WebAssembly is currently providing both full and partial Hot Reload support, dep
...
 
 ### [**iOS, Android, and Catalyst**](#tab/mobile)
 
-Mobile targets are currently using a limited version of XAML Hot Reload and do not support C# Hot Reload until [this dotnet runtime](https://github.com/dotnet/runtime/issues/93860) issue is fixed.
+Mobile targets are currently using a limited version of XAML Hot Reload and do not support C# Hot Reload until [this dotnet runtime](https://github.com/dotnet/android/issues/9120) issue is fixed.
 
 - In Visual Studio, the "Hot Reload on File Save" feature must be disabled to avoid crashing the app. You can find this feature by clicking on the down arrow next to the red flame in the Visual Studio toolbar.
-- In VS, VS Code, and Rider, [C# Hot Reload is not yet supported](https://developercommunity.visualstudio.com/t/net70-iosnet70-android-MetadataUpd/10279604#T-ND10384434)
+- In VS, VS Code, and Rider, [C# Hot Reload is fully not yet supported](https://developercommunity.visualstudio.com/t/net70-iosnet70-android-MetadataUpd/10279604#T-ND10384434)
```
Comment (morning4coffe-dev) [inline doc/articles/features/working-with-xaml-hot-reload.md:282]:
```suggestion
- In VS, VS Code, and Rider, [C# Hot Reload is not yet fully supported](https://developercommunity.visualstudio.com/t/net70-iosnet70-android-MetadataUpd/10279604#T-ND10384434)
```

File: doc/articles/features/working-with-xaml-hot-reload.md [hunk 3]
```diff
@@ -53,7 +53,197 @@ Hot Reload features vary between platforms and IDE, you can check below the list
...
+</table>
+
+---
+
+Legend:
+
+- ✅ Supported
+- 🐞 Supported with the debugger
+- ⌛ Upcoming support
+- 🛜 Supported through [SSH to a mac](xref:Uno.GettingStarted.CreateAnApp.VSCode#debug-the-app)
```
Comment (morning4coffe-dev) [inline doc/articles/features/working-with-xaml-hot-reload.md:235]:
```suggestion
- 🛜 Supported through [SSH to a Mac](xref:Uno.GettingStarted.CreateAnApp.VSCode#debug-the-app)
```

**Review by morning4coffe-dev — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.3

**Pull request merged**